console/src/main/java/org/georchestra/console/ws/backoffice/platform/InfosController.java

```diff
@@ -63,8 +63,14 @@ public class InfosController {
     @Value("${headerScript:https://cdn.jsdelivr.net/gh/georchestra/header@dist/header.js}")
     private String headerScript;
 
+    @Value("${logoUrl:https://www.georchestra.org/public/georchestra-logo.svg}")
```

Comment:
Would be nice to be able to point to a local resource here (independent of the deployment) in the future.